### PR TITLE
new icon: hadoop (original, original-wordmark, plain, plain-wordmark)

### PR DESCRIPTION
**Pull request by MAGistR-bit**
### Apache Hadoop
Opened the pull request (again)

### CODE REVIEW

**Comment by MAGistR-bit** [comment]
@Snailedlt I have opened a new pull request (PR).

**Review by Snailedlt — APPROVED** [review]
Looks good! ✔️ 
Thank you for your contribution, and your patience! 💯 

PS: The bot message saying that 'stroke' should be replaced by 'fill' can be ignored, since the `hadoop-original` icon isn't used on icomoon as explained in this issue: #1325

File: devicon.json [hunk 1]
```diff
@@ -3297,6 +3297,32 @@
...
+                "plain"
+            ]
+        },
+        "color": "#ffff00",
+        "aliases": [
+            {
+                "base": "plain",
+                "alias": "line"
+            }
+        ]
```
Comment (lunatic-fox) [inline devicon.json:3324]:
```suggestion
        "aliases": []
```
I think `plain` version can't be aliased as `line` version. So I suggest this change.

Comment (Snailedlt) [reply devicon.json:3324]: Why can't it? :O

Comment (Snailedlt) [reply devicon.json:3324]: hmm, I don't agrre :/ If anything I think it should qualify as a line version, and possibly not plain version. 
The reason being that the plain version is usually just the original version, but mono-colored and with unified paths. Meanwhile the line version is (in my interpretation) just the inverse of the plain icon. So where there are empty spots in the plain version, there will be lines in the line version, and vice-versa.

I'm curious what others think about this though. 
@Panquesito7 @kilianpaquier Any thoughts on whether or not [this icon](https://github.com/devicons/devicon/pull/1488/files#diff-613b72c291b19bc5cae774fa507bd1c71713db1822749fb5ff8314eb54553ee9) should qualify for plain and/or line versions?

Comment (kilianpaquier) [reply devicon.json:3324]: Hello @Snailedlt, I don't see why it couldn't be aliases as "line" icon. There's only one path, and has only lines ?

Comment (Snailedlt) [reply devicon.json:3324]: Thanks for your input. I agree with this

**Comment by Snailedlt** [comment]
Seems like this PR has become stale. 

@lunatic-fox Feel free to commit the icons you suggested [here](https://github.com/devicons/devicon/pull/1488#issuecomment-1340124922), and take over the PR :)

**Comment by Snailedlt** [comment]
@lunatic-fox Ignore [this](https://github.com/devicons/devicon/pull/1488#issuecomment-1340672909) message 
ref: https://github.com/devicons/devicon/pull/1488#pullrequestreview-1161184500

**Review by Snailedlt — APPROVED** [review]
LGTM! 💯

**Comment by lunatic-fox** [comment]
> @lunatic-fox Ignore [this](https://github.com/devicons/devicon/pull/1488#issuecomment-1340672909) message ref: [#1488 (review)](https://github.com/devicons/devicon/pull/1488#pullrequestreview-1161184500)

Better remove it anyway, don't you think?

**Comment by Snailedlt** [comment]
> > @lunatic-fox Ignore [this](https://github.com/devicons/devicon/pull/1488#issuecomment-1340672909) message ref: [#1488 (review)](https://github.com/devicons/devicon/pull/1488#pullrequestreview-1161184500)
> 
> Better remove it anyway, don't you think?

No need imo. Well if it gets the file size down without changing the icon, then sure, otherwise we can just keep it :)

**Comment by lunatic-fox** [comment]
> > > @lunatic-fox Ignore [this](https://github.com/devicons/devicon/pull/1488#issuecomment-1340672909) message ref: [#1488 (review)](https://github.com/devicons/devicon/pull/1488#pullrequestreview-1161184500)
> > 
> > 
> > Better remove it anyway, don't you think?
> 
> No need imo. Well if it gets the file size down without changing the icon, then sure, otherwise we can just keep it :)

It's a little stroke from original icons. Don't know why is there, because it doesn't appear, but in order to prevent any resize bug (like it for some reason don't resize with the icon) I'll remove it really fast. 🙂

**Review by Snailedlt — APPROVED** [review]

**Pull request merged**